### DISCUSSION TITLE
custom install dir

### DISCUSSION THREAD

**Discussion by R08UST** [wbthomason/packer.nvim#309] [Q&A] [answered]
Hi, Is there any solution that install packer and other plugins in the specified dir?    

just like Dein:
    
  set runtimepath+=~/.config/nvim/dein/repos/github.com/Shougo/dein.vim
  call dein#begin('~/.config/nvim/dein')
  call dein#add('~/.config/nvim/dein/repos/github.com/Shougo/dein.vim')
  " dein plugins

Thank you.

**Comment by nanotee** [comment, marked as answer]
I believe the `package_root` option is what you're looking for (see `:help packer-configuration`):

```lua
local packer = require('packer')

packer.startup{function(use)
  -- plugin specification goes here
end,
config = {
  -- The root has to be a directory named "pack"
  package_root = vim.fn.stdpath('config') .. '/packer/pack',
}}
```

You'll have to add the string to `packpath` as well:

```lua
vim.o.packpath = vim.o.packpath .. ',' .. vim.fn.stdpath('config') .. '/packer' -- without "/pack"
```

**Reply by R08UST** [reply]
thx! it works
